pkgs/servers/x11/xorg/overrides.nix

```diff
@@ -829,7 +829,6 @@ self: super:
   });
 
   xf86videovmware = super.xf86videovmware.overrideAttrs (attrs: {
-    buildInputs = attrs.buildInputs ++ [ mesa ];
```

Comment:
Why this change? Gallium? Could be noted in commit message.

Reply:
Will split into another commit.